typescript/packages/plugins/modespray/src/parameters.ts

```diff
@@ -0,0 +1,16 @@
+import { z } from "zod";
+
+export const DisperseEtherParameters = z.object({
+    recipients: z
+        .array(z.string().regex(/^0x[a-fA-F0-9]{40}$/, "Invalid address"))
```

Comment:
Needs to have a description for the LLM to know what this parameter is :)

Name is self explanatory, but better to have it for "less capable" models